### PR TITLE
Update zemu zxlib

### PR DESCRIPTION
**Pull request by ftheirs**
Update zxlib
Update zemu

### CODE REVIEW

File: deps/ledger-zxlib/app/common/view_s.c [hunk 1]
```diff
@@ -189,12 +192,42 @@ void h_review_button_both() {
...
     if (vlen > MAX_CHARS_PER_VALUE2_LINE - 1) {
         snprintf(viewdata.value2, MAX_CHARS_PER_VALUE2_LINE, "%s", viewdata.value + MAX_CHARS_PER_VALUE_LINE);
         viewdata.value[MAX_CHARS_PER_VALUE_LINE] = 0;
     }
 }
+void splitValueAddress() {
+    uint8_t len = MAX_CHARS_PER_VALUE_LINE;
+    bool exceeding_max = exceed_pixel_in_display(len);
+    while(exceeding_max) {
+        len--;
```
[the remark this answers is not in the document]
Comment (ftheirs) [reply deps/ledger-zxlib/app/common/view_s.c]: It's an uint8 so it won't be negative but an infinitive loop would be the problem. Fixed!

File: deps/ledger-zxlib/app/common/view_s.c [hunk 2]
```diff
@@ -189,12 +192,42 @@ void h_review_button_both() {
...
+        snprintf(viewdata.value2, MAX_CHARS_PER_VALUE2_LINE, "%s", viewdata.value + len);
+        viewdata.value[len] = 0;
+    }
+}
+
+max_char_display get_max_char_per_line() {
+    uint8_t len = MAX_CHARS_PER_VALUE_LINE;
+    bool exceeding_max = exceed_pixel_in_display(len);
+    while(exceeding_max) {
+        len--;
```
[the remark this answers is not in the document]
Comment (ftheirs) [reply deps/ledger-zxlib/app/common/view_s.c]: Save comment as previous part

File: deps/ledger-zxlib/src/zxutils_ledger.c [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+   7 << 4 |  6,   /* code 007F */
+};
+
+unsigned short zx_compute_line_width_light(const char* text, unsigned char text_length) {
+  char current_char;
+  unsigned short line_width = 0;
+
+  // We parse the characters of the input text on all the input length.
+  while (text_length--) {
+    current_char = *text;
```
[the remark this answers is not in the document]
Comment (ftheirs) [reply deps/ledger-zxlib/src/zxutils_ledger.c:124]: A null check was added here!

**Comment by jleni** [comment]
merged into dev, rebased and opened another PR

**Pull request closed without merging**